glibc/Dockerfile.builder

```diff
@@ -173,3 +173,5 @@ RUN set -eux; \
 RUN cp -L /etc/resolv.conf rootfs/etc/; \
 	chroot rootfs /bin/sh -xec 'nslookup google.com'; \
 	rm rootfs/etc/resolv.conf
+
+# vim:ft=Dockerfile
```

Comment:
I don't think this is actually working correctly (especially as the highlighting appears to be wrong here).  I believe the following is what you intended:

```suggestion
# vim:set ft=dockerfile:
```

Reply:
(And would then be similar to other repositories we maintain where we've done something similar; for example: https://github.com/docker-library/drupal/blob/1781a47e9d6161e504787e5d2b26fae6117a4751/Dockerfile-debian.template#L77)

Reply:
Both are valid (see for example https://github.com/jane/dockerfiles/blob/master/node-base#L14), but I'll change them to your suggestion for consistency with other repos.